Mathlib/Analysis/SpecialFunctions/Pow/Real.lean

```diff
@@ -255,6 +255,22 @@ theorem ofReal_cpow_of_nonpos {x : ℝ} (hx : x ≤ 0) (y : ℂ) :
     arg_ofReal_of_nonneg (neg_nonneg.2 hx), ofReal_zero, zero_mul, add_zero]
 #align complex.of_real_cpow_of_nonpos Complex.ofReal_cpow_of_nonpos
 
+lemma cpow_ofReal (x : ℂ) (y : ℝ) :
+    x ^ y = ↑((abs x) ^ y) * (Real.cos (arg x * y) + Real.sin (arg x * y) * I) := by
```

Comment:
Can you add explicit casts here? This looks very fragile WRT the elaboration of `^`

Reply:
```suggestion
    x ^ (y : ℝ) = ↑(abs x ^ y) * (Real.cos (arg x * y) + Real.sin (arg x * y) * I) := by
```